freddytoolapi/Freddy.UnitTests/Utilities/HttpClientExtensions.cs

```diff
@@ -12,13 +12,21 @@ public static class HttpClientExtensions
         {
             PropertyNamingPolicy = JsonNamingPolicy.CamelCase
         };
-        
+
         public static async Task<TObject> GetObjectAsync<TObject>(this HttpClient client, string url)
         {
             var response = await client.GetStreamAsync(url);
-            return await JsonSerializer.DeserializeAsync<TObject>(response, Options);
+
+            try
```

Comment:
stale tu mas try catch :D

Reply:
som si nevsimol ze to je na dvoch miestach pri Unit aj pri Integration